tests/utils/test_file_utils.py

```diff
@@ -75,6 +82,7 @@ def test_render_and_merge_yaml(tmpdir):
             test_2: {{ TEST_VAR_1 }}
             test_3: {{ TEST_VAR_2 }}
             """
+            + f"test_4: {{{{ '{json_file}' | from_json }}}}"
```

Comment:
```suggestion
            + f"test_4: {{{{ '{json_file}' | from_json }}["key"]}}"
```

Just a nit, it would be nice to update the test to reflect typical usage (accessing a field within the loaded JSON). Not sure I got the curly-brace escaping right :)

Reply:
Sorry this suggestion doesn't work.